src/test/java/uk/gov/pay/webhooks/webhook/resource/WebhookResourceTest.java

```diff
@@ -283,7 +290,21 @@ public void getWebhooksRequestWithLiveParamOnlyShould400() throws JsonProcessing
         Map<String, String> responseBody = objectMapper.readValue(response.readEntity(String.class), new TypeReference<>() {
         });
         assertThat(response.getStatus(), is(400));
-        assertThat(responseBody.get("message"), is("either service_id or override_service_id_restriction query parameter must be provided"));
+        assertThat(responseBody.get("message"), is("[service_id, gateway_account_id] or override_service_id_restriction query parameter must be provided"));
+    }
+
+    @Test
+    public void getWebhooksRequestWithLiveAndServiceWithoutGatewayAccoutnIdShould400() throws JsonProcessingException {
```

Comment:
```suggestion
    public void getWebhooksRequestWithLiveAndServiceWithoutGatewayAccountIdShould400() throws JsonProcessingException {
```

could we add a similar tests for `/v1/webhook/{webhookExternalId}`?